docs/changehistory/NextVersion.md

```diff
@@ -143,3 +145,7 @@ To make it easier to use async APIs while exporting a schema, [IModelExportHandl
 [IModelTransformer.shouldExportSchema]($backend) now gets a [SchemaKey]($ecschema-metadata) schema key as argument, instead of a full [Schema]($ecschema-metadata). If you
 need to check the full schema, return `true` in shouldExportSchema and in [IModelExportHandler.onExportSchema]($backend), you can use the schema object to check and then
 return early.
+
+### [@bentley/imodeljs-backend](https://www.itwinjs.org/reference/ecschema-metadata/)
+
+To reduce the size and limit the scope of the APIs available in the ecschema-metadata package, all APIs associated with EC Schema editing and validation have been moved to the [@bentley/ecschema-editing](https://www.itwinjs.org/reference/ecschema-editing/) package. This includes all source code under the [Validation](https://www.itwinjs.org/reference/ecschema-metadata/) and [Editing](https://www.itwinjs.org/reference/ecschema-metadata/editing/) folders. All corresponding @beta types defined in the ecschema-metadata package have been deprecated.  All @alpha types have been removed from the ecschema-metadata package.
```

Comment:
Say that the is the first step in our schema editing story.  Link to the discussions page about it.  

Make a bullet list of what has moved/changed so it's easier to see with a quick glance

Reply:
There quite a few types.  Not sure if this is what you are looking for...but have a look.